### PR TITLE
Fix some typos in TRPL

### PR DESCRIPTION
**Pull request by ykomatsu**

### CODE REVIEW

File: src/doc/trpl/error-handling.md [hunk 1]
```diff
@@ -2057,7 +2057,7 @@ so. This can be a little clumsy, especially if you intend for the program to
 be used in shell scripts.
 
 So let's start by adding the flags. Like before, we need to tweak the usage
-string and add a flag to the Option variable. Once were done that, Getopts does the rest:
+string and add a flag to the Option variable. Once we're done that, Getopts does the rest:
```
Comment (alexcrichton) [inline src/doc/trpl/error-handling.md]: I think this may actually want to be "we've" ?

Comment (steveklabnik) [inline src/doc/trpl/error-handling.md]: correct!

Comment (ykomatsu) [inline src/doc/trpl/error-handling.md]: Thank you @alexcrichton and @steveklabnik.

I corrected this.

**Pull request merged**